src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -97,17 +154,14 @@ def __init__(self) -> None:
             "   AND vt.static_version_key = sr.static_version_key"
             " WHERE "
             "   sr.route_type < 2"
-            "   AND ("
-            "       ve.canonical_stop_sequence > 1"
-            "       OR ve.canonical_stop_sequence IS NULL"
-            "   )"
+            "   AND drop_join.drop_flag IS NULL "
             "   AND ("
             "       ve.vp_stop_timestamp IS NOT null"
             "       OR ve.vp_move_timestamp IS NOT null"
             "   )"
             "   %s"
             " ORDER BY "
-            "   ve.service_date, vt.route_id, vt.direction_id, vt.vehicle_id, vt.start_time"
+            "   ve.service_date, vt.vehicle_id, vt.start_time"
```

Comment:
was the order change requested?

Reply:
It wasn't requested, but we're only doing this to reduce the parquet file size and dropping the two SORT field fields gave us a speed up in query time with almost no change in file size.